_posts/2019-07-24-lts-plan.md

```diff
@@ -0,0 +1,19 @@
+---
+layout: post
+title: Introducing our Long Term Support plan
+video: 2HuiH-0R6a0
+---
+
+Today we're formally announcing our [Long Term Support plan](https://github.com/twbs/release), a documented approach aimed at strengthening the stability and frequency of releases. As part of this initiative, each major version of Bootstrap will receive at least six months of support after it is retired, followed by six months of critical bug fixes and security updates.
+
+Starting today, Bootstrap 3 will move to end of life, and will no longer receive critical security updates.
+
+Bootstrap 4 moved to Long Term Support release on ____ and will no longer receive new features. It will continue to receive bug fixes, security updates, and documentation updates.
+
+Bootstrap 5 is under active development. You can follow our progress [on GitHub](https://github.com/twbs/bootstrap).
+
+A special thanks to [@xhmikosr](https://github.com/xhmikosr) for their tireless work in pushing us forward.
```

Comment:
Perhaps this should be rephrased? I don't want @Johann-S or @MartijnCuppens to feel left out.

Reply:
If it comes to pushing us to to get things done, credits go to you, @XhmikosR. I don't mind not getting mentioned here.

Reply:
not really active currently, so that's ok to me too 👍

Reply:
maybe we should reword a bit if just speak about @XhmikosR 🤔
`special thanks to @XhmikosR for his tireless work`